libraries/AP_NavEKF2/AP_NavEKF2_Control.cpp

```diff
@@ -299,20 +299,20 @@ void NavEKF2_core::setAidingMode()
             bool canUseExtNav = readyToUseExtNav();
             // We have commenced aiding and GPS usage is allowed
             if (canUseGPS) {
-                gcs().send_text(MAV_SEVERITY_INFO, "EKF2 IMU%u is using GPS",(unsigned)imu_index);
+                gcs().send_text(MAV_SEVERITY_INFO, "EKF2 %u (g%u a%u) is using GPS",(unsigned)core_index, (unsigned)gyro_index, (unsigned)accel_index);
```

Comment:
I'd remove this msg change

Reply:
A consistent message suffix like "EKF2/n ......" where n is the core index is preferred so I think some change to these messages is required.